Common/UnitDefinitions/TemperatureDelta.json

```diff
@@ -20,11 +20,12 @@
       "PluralName": "DegreesCelsius",
       "FromUnitToBaseFunc": "x",
       "FromBaseToUnitFunc": "x",
-      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Hecto", "Kilo" ],
+      "Prefixes": [ "Milli" ],
```

Comment:
I cannot find on Google clear examples of whether the prefixes nano, micro, centi, deci, hecto, and kilo are used in the real world. Leaving it for future developers to add them if/when the need for them is clear.

Reply:
Fine by me, I also prefer that units are actually used by someone.